### DISCUSSION TITLE
How to use react-error-boundary with tRPC ?

### DISCUSSION THREAD

**Discussion by jeanhdev** [trpc/trpc#3997] [Q&A]
When an errors comes up with tRPC - it does not trigger the error boundary... (via `react-error-boundary`).

Anybody has an idea on how to implement client error handling ?

**Comment by Nick-Lucas** [comment]
This is more of a react-query question, the tRPC react-query package is just a shim around it: https://tanstack.com/query/v4/docs/react/reference/QueryErrorResetBoundary
